services/elastic/docker-compose.yaml

```diff
@@ -0,0 +1,31 @@
+version: '2.2'
+
+services:
+  elastic:
+    image: docker.elastic.co/elasticsearch/elasticsearch:8.13.1
+    volumes:
+      - esdata:/usr/share/elasticsearch/data
+    labels:
+      - traefik.http.routers.elastic.rule=Host(`elastic.localhost`)
+      - traefik.http.services.elastic.loadbalancer.server.port=9200
+    mem_limit: 1073741824
```

Comment:
is this needed?

Reply:
the elastic forwarding isn't needed actually, I'll remove it

Reply:
see below